charts/k8s-monitoring/README.md

```diff
@@ -242,6 +250,7 @@ The Prometheus and Loki services may be hosted on the same cluster, or remotely
 | metrics.kubelet.scrapeInterval | string | 60s | How frequently to scrape metrics from the Kubelet. Overrides metrics.scrapeInterval |
 | metrics.kubernetesMonitoring.enabled | bool | `true` | Report telemetry about this Kubernetes Monitoring chart as a metric. |
 | metrics.node-exporter.allowList | list | See [Allow List for Node Exporter](#allow-list-for-node-exporter) | The list of Node Exporter metrics that will be scraped by the Agent |
+| metrics.node-exporter.cardinalityImprovements.dropTempFSMetrics | bool | `true` | Drop node filesystem metrics with tempfs |
```

Comment:
I like the more configurable version in `metrics.cadvisor.cardinalityImprovements.keepPhysicalFilesystemDevices`, rather than this simpler option.

Reply:
Done